docs/server/source/server-reference/bigchaindb-cli.md

```diff
@@ -83,107 +83,113 @@ configuration file as documented under
 [SUCCESS] Submitted proposal with id: 04a067582cf03eba2b53b82e4adb5ece424474cbd4f7183780855a93ac5e3caa
 ```
 
-A successful execution of the above command **doesn't** imply that the validator set will be immediately updated but rather it means the proposal has been succcessfully accepted by the network. Once the `election_id` has been generated the node operator should share this `election_id` with other validators in the network and urge them to approve the proposal. Note that the node operator should themsleves also approve the proposal.
+A successful execution of the above command does not imply the validator set has been updated but rather the proposal has been accepted by the network.
+Once `election_id` has been generated, the proposer should share it with other validators of the network (e.g. via email) and ask them to approve the proposal.
+
+Note that election proposers do not automatically approve elections by proposing them.
 
+For more details about how validator set changes work, refer to [BEP-21](https://github.com/bigchaindb/BEPs/tree/master/21).
 
-###### election new migration
+###### election new chain-migration
 
-Create an election to halt block production, to allow for a version change across breaking changes.
+Create an election to halt block production, to coordinate on making a Tendermint upgrade with a backwards-incompatible chain.
 
 
 ```bash
-$ bigchaindb election new migration --private-key PATH_TO_YOUR_PRIVATE_KEY
-[SUCCESS] Submitted proposal with id: <election_id>
+$ bigchaindb election new chain-migration --private-key <path-to-theprivate-key>
 ```
 
-- `--private-key`: The path to Tendermint's private key which can be generally found at `/home/user/.tendermint/config/priv_validator.json`.
+- `<path-to-the-private-key>` is the path to the private key of the validator who proposes the election. Tendemint places it at  `.tendermint/config/priv_validator.json`.
+
 
-Example usage,
+Example:
 
 ```bash
 $ bigchaindb election new migration --private-key /home/user/.tendermint/config/priv_validator.json
 [SUCCESS] Submitted proposal with id: 04a067582cf03eba2b53b82e4adb5ece424474cbd4f7183780855a93ac5e3caa
 ```
 
-**NOTE** `migration` elections will halt block production at whichever blockheight they are approved. Once the election is concluded, the validators will need to restart their systems with a new `chain_id` to resume normal operations.
+Concluded chain migration elections halt block production at whichever block height they are approved.
+Afterwards, validators are supposed to upgrade Tendermint, set new `chain_id`, `app_hash`, and `validators` (to learn these values, use the [election show](#election-show) command) in `genesis.json`, make and save a MongoDB dump, and restart the system.
 
 
+For more details about how chain migrations work, refer to [BEP-42](https://github.com/bigchaindb/BEPs/tree/master/42).
```

Comment:
Maybe we could be a bit more helpful and say "refer to [Type 3 scenarios in BEP-42](https://github.com/bigchaindb/BEPs/tree/master/42#type-3-scenarios)."